contracts/utils/cryptography/SignatureChecker.sol

```diff
@@ -0,0 +1,21 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity >=0.6.0 <0.9.0;
+
+import "./ECDSA.sol";
+import "../Address.sol";
+import "../../interfaces/IERC1271.sol";
+
+library SignatureChecker {
```

Comment:
Missing docs! We should make particular emphasis on why the function is called "...Now".